### PR TITLE
multiple config changes for raw3-4

### PR DESCRIPTION
**Pull request by abubeck**

### CODE REVIEW

**Comment by NadiaHG** [comment]
Visual Review: OK

**Comment by floweisshardt** [comment]
therre is a lot of mess regarding tabs and spaces. @abubeck please fix it before merging.

**Comment by abubeck** [comment]
Changed tabs

File: cob_bringup/drivers/base.launch [hunk 1]
```diff
@@ -19,15 +19,14 @@
 		
 		<!-- start undercarriage ctrl -->
 		<node pkg="cob_undercarriage_ctrl" type="cob_undercarriage_ctrl_node" name="undercarriage_ctrl_node" respawn="false" output="screen">
-			<remap from="command" to="command_direct"/>
+		<remap from="command" to="command_direct"/>
```
Comment (floweisshardt) [inline cob_bringup/drivers/base.launch]: still lacking one more tab here

**Comment by abubeck** [comment]
Next try :)

File: cob_bringup/drivers/base.launch [hunk 2]
```diff
@@ -20,14 +20,13 @@
 		<!-- start undercarriage ctrl -->
 		<node pkg="cob_undercarriage_ctrl" type="cob_undercarriage_ctrl_node" name="undercarriage_ctrl_node" respawn="false" output="screen">
 			<remap from="command" to="command_direct"/>
+			<param name="boolCtrl" value="1"/>
```
Comment (floweisshardt) [inline cob_bringup/drivers/base.launch]: will this influence other robots?

Comment (abubeck) [inline cob_bringup/drivers/base.launch]: This shouldn't be used on other robots as the parameter is only present in the raw3-4 undercarriage ctrl. The other robots should just ignore it.

**Comment by abubeck** [comment]
Updated with upstream changes.

**Comment by floweisshardt** [comment]
visual review :ok:

**Comment by NadiaHG** [comment]
Visual Review :ok:

**Pull request merged**